src/viam/sdk/components/private/camera_server.cpp

```diff
@@ -114,6 +114,7 @@ ::grpc::Status CameraServer::GetProperties(
         *response->mutable_intrinsic_parameters() =
             Camera::to_proto(properties.intrinsic_parameters);
         response->set_supports_pcd(properties.supports_pcd);
+        response->set_frame_rate(properties.frame_rate);
```

Comment:
is this the correct function "set_frame_rate()" because it comes from Property from camera and not getPropertiesResponse

Reply:
Yep! In this method we are populating a `GetPropertiesResponse` using data from the camera's `Properties`